File: ejecucion_1/APPS/ImageButton/app/src/main/java/com/example/imagebutton/MainActivity.java
package com.example.imagebutton;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    public void MensajeGato(View view){
        Toast.makeText(this, "Este es un gatito", Toast.LENGTH_LONG).show();
    }
    public void MensajePerro(View view){
        Toast.makeText(this, "Este es un perrito", Toast.LENGTH_LONG).show();
    }
}

File: Bootcamp_Android_Studio_Basico/APPS/APP_K1/app/src/main/java/com/example/app_k1/MainActivity.kt
package com.example.app_k1

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        cualquiercosa()
        aca_numeros()
    }
    private fun cualquiercosa(){
        var variable1="primera frase con kotlin";
        println(variable1);

        var variable2= 2;
        println(variable2);

        val constante1= "esta no cambia";
        println(constante1);
        //val constante1="quiero que cambie";

        val constante2= variable1;
        println(constante2);

    }
    private fun aca_numeros(){
        val frase1 = "hello";
        val frase2 = "como estas?";
        val frase3 = frase1 +""+ frase2;
        println(frase3);

        val num1 = 32;
        val num2 = 2.63f;
        val num3 = 3.9;
        val num4 = num1 + num2 + num3;
        println(num4);

        val booleano1 = true;
        val booleano2 = false;
        println(booleano1 == booleano2);
        print(booleano1 && booleano2); //suma logica;

    }
}